client/src/main/java/com/arcbees/beeshop/client/application/home/HomeView.ui.xml

```diff
@@ -97,9 +102,13 @@
         </section>
 
         <section class="{res.style.stripe} {page.style.welcome}">
-            <h2>Welcome to the Beestore</h2>
-            <p>The Beestore is a study case from Arcbees
-                <span>to show our hive of products and branded gears. Enjoy!</span>
+            <p>
+                <ui:msg description="case study">The Beestore is a case study from Arcbees</ui:msg>
```

Comment:
Maybe you could move those to a Constants interface. I believe you could use them with an `ui:with` field